src/modules/transactions/actions/construct-transaction.js

```diff
@@ -139,7 +139,10 @@ export function constructTransferTransaction(log, address) {
 }
 
 export const constructCancelOrderTransaction = (trade, marketId, marketType, description, outcomeId, outcomeName, minPrice, maxPrice, status) => (dispatch, getState) => {
-  const displayPrice = augur.trading.denormalizePrice({ normalizedPrice: trade.price, minPrice, maxPrice })
+  const { orderBooks } = getState()
+  const type = trade.orderType === '0' ? TYPES.BUY : TYPES.SELL
+  const price = !trade.price ? orderBooks[marketId][outcomeId][type][trade.orderId].price : trade.price
```

Comment:
I'm not sure I understand what's going on here, is `price` not always included with the trade?  If not, suggest just including it with augur-node's payload.

Note that this code is all significantly modified on the rollback branch, so it's preferable not to also make changes to it here.

Reply:
ok, then only changing listen-to-update with property change and re-ordering method calls will fix the issue.